blocks/card/card.css

```diff
@@ -0,0 +1,86 @@
+.card-container {
+  container-type: inline-size;
+  container-name: card-container;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #dadada;
+  background-color: var(--color-background);
+  display: flex;
+  flex-direction: column;
+  border-radius: 1rem;
+  box-shadow: var(--drop-shadow-emphasized);
+  height: auto;
+}
+
+.card__body {
+  padding: 1.5rem 1rem;
+  box-sizing: border-box;
+}
+
+.card__title {
+  color: var(--color-text-dark);
+  font-size: var(--spectrum-title-size-m);
```

Comment:
Can we use the title class instead of the variable, and then if it changes we can adjust it at larger container sizes? The base style would be for the small card and might not need an override.